### PR TITLE
add tidb_opt_prefer_range_scan variable

### PR DESCRIPTION
**Pull request by xiaodong-ji**
### What is changed, added or deleted?
add tidb_opt_prefer_range_scan variable description for system-variables.md.

### Which TiDB version(s) do your changes apply to? 

- [x] master (the latest development version)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?
https://github.com/pingcap/tidb/pull/18996

### CODE REVIEW

**Review by winoros — COMMENTED** [review]
lgtm

File: system-variables.md [hunk 1]
```diff
@@ -931,3 +931,35 @@ set tidb_slow_log_threshold = 200;
 - 作用域：SESSION | GLOBAL
 - 默认值：ON
 - 这个变量用于控制计算窗口函数时是否采用高精度模式。
+
+### `tidb_opt_prefer_range_scan`
+
+- 作用域：SESSION
+- 默认值：0
+- 这个变量用来设置优化器总是偏好索引扫描而不是全表扫描。
```
Comment (TomShawn) [inline system-variables.md:939]: 可否说明一下如何开启这个功能？是将变量值设为 `1` 吗？

Comment (xiaodong-ji) [reply system-variables.md:939]: 是的，一般参数都是通过设置 0 来关闭功能，通过 设置 1 来开启功能。

Comment (TomShawn) [reply system-variables.md:939]: [L954](https://github.com/pingcap/docs-cn/pull/4745/files#diff-ec2cbb115cc58fe3afe2862eb5148732d323cbf7ae35dea97bd381a1f8d0e8a0R954) 用的是 `ON`，能辛苦确认一下是 `1` 还是 `ON` 呢？谢谢~

Comment (xiaodong-ji) [reply system-variables.md:939]: 1 和 ON 都是一样的，示例中我修改成 1 了

Comment (TomShawn) [reply system-variables.md:939]: 好的，谢啦~

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**